### PR TITLE
Update Electron to 21 + Node to 16

### PR DESCRIPTION
**Pull request by TheCleric**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Whatsapp Web ([HN thread](https://news.ycombinator.com/item?id=10930718)). Nativ
 Install Nativefier globally with `npm install -g nativefier` . Requirements:
 
 - macOS 10.10+ / Windows / Linux
```
Comment (mattruzzi) [inline README.md:34]: Electron 20 removes support for  macOS 10.11 and 10.12 so the minimum version is now macOS 10.13.
https://www.electronjs.org/docs/latest/breaking-changes#removed-macos-1011--1012-support

Comment (TheCleric) [reply README.md:34]: Great catch. Thanks for the attention to @mattruzzi .

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Use Node.js 19
```
Comment (mattruzzi) [inline .github/workflows/ci.yml:22]: This should be changed to "Use Node.js 20"

Comment (TheCleric) [reply .github/workflows/ci.yml]: Good catch. Fixed.

**Review by mattruzzi — COMMENTED** [review]
The latest version of Firefox is now 115.0.3.

**Pull request merged**